build.gradle

```diff
@@ -18,6 +18,15 @@ allprojects {
     }
 }
 
+ext {
+    compileSdkVersion = 25
```

Comment:
Having these in the root project means they only have to be updated in a single location.